base-api.Tests/V1/Helper/DatabaseEntityHelper.cs

```diff
@@ -0,0 +1,32 @@
+using Bogus;
+using base_api.V1.Domain;
+
+namespace UnitTests.V1.Helper
+{
+    public static class DatabaseEntityHelper
+    {
+        public static DatabaseEntity CreateDatabaseEntity()
+        {
+            return CreateDatabaseEntityFrom(EntityHelper.CreateExampleEntity());
+        }
+
+        public static DatabaseEntity CreateDatabaseEntityFrom(ExampleEntity exampleEntity)
+        {
+            var faker = new Faker();
+            var databaseEntity = CopyExampleEntityFields(exampleEntity);
+            databaseEntity.Id = faker.Random.Int();
```

Comment:
Is this mapping the fields from the domain object to the database entity and then reassigning all the fields to random ones anyway?

Reply:
Doh, yes. Leftover from the UH-specific domain & database code, which included tons of properties and only the ID was randomized, IIRC. Removing this now, thanks for flagging 👍